source/microbit/help.c

```diff
@@ -68,58 +69,109 @@ STATIC const mp_doc_t help_table_types[] = {
+    {&microbit_p7_obj, "micro:bit's pin 7 on the gold edge connector.\n"},
+    {&microbit_p8_obj, "micro:bit's pin 8 on the gold edge connector.\n"},
+    {&microbit_p9_obj, "micro:bit's pin 9 on the gold edge connector.\n"},
+    {&microbit_p10_obj, "micro:bit's pin 10 on the gold edge connector.\n"},
+    {&microbit_p11_obj, "micro:bit's pin 11 on the gold edge connector.\n"},
+    {&microbit_p12_obj, "micro:bit's pin 12 on the gold edge connector.\n"},
+    {&microbit_p13_obj, "micro:bit's pin 13 on the gold edge connector.\n"},
+    {&microbit_p14_obj, "micro:bit's pin 14 on the gold edge connector.\n"},
+    {&microbit_p15_obj, "micro:bit's pin 15 on the gold edge connector.\n"},
+    {&microbit_p16_obj, "micro:bit's pin 16 on the gold edge connector.\n"},
+    {&microbit_p19_obj, "micro:bit's pin 19 on the gold edge connector.\n"},
+    {&microbit_p20_obj, "micro:bit's pin 20 on the gold edge connector.\n"},
+    {&microbit_pin_write_digital_obj, "micro:bit, write_digital(choice) to the pin. You have two 'choice' values,\n0 (lo) or 1 (hi).\n"},
+    {&microbit_pin_read_digital_obj, "micro:bit, read_digital() value from the pin as either 0 (lo) or 1 (hi).\n"},
+    {&microbit_pin_write_analog_obj, "micro:bit, write_analog(value) to the pin. You can use a whole number between\n0 and 255 as the value.\n"},
+    {&microbit_pin_read_analog_obj, "micro:bit, read_analog() value from the pin. Wow, analog has lots of values\n(0 - 65535). Digital has only 0 and 1.\n"},
+    {&microbit_pin_is_touched_obj, "If pin is_touched() on micro:bit, return True. If nothing is touching the pin,\nreturn False.\n"},
+    // I2C
+    {&microbit_i2c_obj, "Communicate with named devices connected to micro:bit using the I/O pins.\n"},
+    {&microbit_i2c_read_obj, "Use read(addr, n) to read n bytes from the device whose address is addr.\n"},
+    {&microbit_i2c_write_obj, "Use write(addr, buf) to write the buffer 'buf' to the device whose address\nis addr.\n"},
+    // Image
+    {&microbit_image_type, "Create and use built-in IMAGES to show on the display. Use:\nImage(\n  '09090:'\n  '99999:'\n  '99999:'\n  '09990:'\n  '00900:')\n...to make a new 5x5 heart image. Numbers go from 0 (off) to 9 (brightest). Note\nthe colon ':' to set the end of a row.\n"},
+    {&microbit_image_width_obj, "Return the width of the image in pixels.\n"},
+    {&microbit_image_height_obj, "Return the height of the image in pixels.\n"},
+    {&microbit_image_get_pixel_obj, "Use get_pixel(x, y) to return the image's brightness at LED pixel (x,y).\nBrightness can be from 0 (LED is off) to 9 (maximum LED brightness).\n"},
+    {&microbit_image_set_pixel_obj, "Use set_pixel(x, y, b) to set the LED pixel (x,y) in the image to brightness\n'b' which can be set between 0 (off) to 9 (full brightness).\n"},
+    {&microbit_image_shift_left_obj, "Use shift_left(i) to make a copy of the image but moved 'i' pixels to the left.\n"},
```

Comment:
Indeed it does, for some reason at the time of writing I though `left` and `right` had the same number of letters, and I even counted them, which makes it even more embarrassing.